.github/workflows/mirror_gitee.yml

```diff
@@ -0,0 +1,18 @@
+name: Mirror to Gitee Repo
+
+on: [ push, delete, create ]
+
+# Ensures that only one mirror task will run at a time.
+concurrency:
+  group: git-mirror
+
+jobs:
+  git-mirror:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: wearerequired/git-mirror-action@v1
+        env:
+          SSH_PRIVATE_KEY: ${{ secrets.SSH_PRIVATE_KEY }}
```

Comment:
Have you added secrets? I suggest add into **organization secrets**.

Reply:
Also, have you added the key into Gitee?

Reply:
I'm retracting this. I think there might be a better solution.